config/develop/namespaced/sqs-queue.yaml

```diff
@@ -0,0 +1,13 @@
+template:
```

Comment:
Likewise for the s3-event-config-lambda

Reply:
I guess when I originally created this PR, I wasn't sure if we were going to push this right away or leave it in dev branch to avoid messing with production. Based on our recent discussion, it seems like we should be okay to also have it in prod so will do.

Reply:
Sure. It will exist independently of everything else for now. We do modify the behavior of the event config lambda, but we aren't using it right now so it shouldn't matter.